src/components/collaborator-socio-card.tsx

```diff
@@ -0,0 +1,95 @@
+interface Cardprops {
+  type: "colaborador" | "socio";
```

Comment:
I don't really like how these values are `string`s and in portuguese.
Maybe this can be an `enum` with values `Collaborate` and `Membership`?

Reply:
Agreed, I had also suggested "JoinUsCard" earlier but I think that's better.